src/common.cpp

```diff
@@ -234,7 +234,7 @@ const char *handler2str(fd_block_handler_t type) {
     static const char *s_fds_handle_desc[FD_HANDLE_MAX] = { "recvfrom", "recvfrom", "select"
 #ifndef __windows__
                                                             ,
-                                                            "poll", "epoll",
+                                                            "poll", "epoll", "kqueue",
```

Comment:
I guess that `kqueue` should be under `defined(__FreeBSD__) || defined(__APPLE__)`